Cargo.lock

```diff
@@ -1498,6 +1499,17 @@ dependencies = [
  "zeroize",
 ]
 
+[[package]]
+name = "crypto-bigint"
```

Comment:
Also I can see we're using two different versions of `crypto-bigint` library. "0.5.5" and `0.6.0-..`

Reply:
Good catch, I need to figure out why